spec/index.bs

```diff
@@ -15811,7 +15812,7 @@ This feature adds the following [=optional API surfaces=]:
 <span id=dom-gpufeaturename-texture-compression-bc></span>
 </h3>
 
-Allows for explicit creation of textures of BC compressed formats. Supports both 2D and 3D textures.
+Allows for explicit creation of textures of BC compressed formats. Only supports 2D textures.
```

Comment:
We should note that these two features will always come together, in three places:
- here
- in the definition for texture-compression-bc-sliced-3d
- in the "Adapter Capability Guarantees" section

Reply:
@kainino0x thank you, I tried to incorporate in the new commit. I tried to keep it simple but in case it is not suitable, I'd appreciate guidance.